src/foundryvtt-gmScreen.ts

```diff
@@ -89,3 +89,31 @@ Hooks.on('renderJournalDirectory', (app, html, data) => {
     _addGmScreenButton(html);
   }
 });
+
+Hooks.on('renderGmScreenApplication', (app,html,data) => {
+  Hooks.callAll("gmScreen", app, true);
+});
+
+Hooks.on('closeGmScreenApplication', (app,html,data) => {
+  Hooks.callAll("gmScreen", app, false);
+});
+
+Hooks.on('renderGmScreen', (render) => { 

```

Comment:
Is this intended to provide a way for other modules to open/close the GM Screen whether using Drawer or not Drawer? If so, I would rather modify the `window[MODULE_ID].toggleGmScreenVisibility` function added to provided [here](https://github.com/ElfFriend-DnD/foundryvtt-gmScreen/blob/main/src/foundryvtt-gmScreen.ts#L54).

Reply:
Actually, I've been hearing lately that `window` might not be the best place to provide such an API, rather putting an object on `globalThis` would be better. Up to you if you want to make that change in this PR or keep it on `window`.